# location:
#   attack_place: [1,0]
#   home: [0,0]
#   base: [0,0,90]
#   recover_place: [-4.4,-6.3]
#   home_xunluo: [0,0]
#   home_xunluo2: [1,0]
#   location1: [7.17,2.82,30]
#   location2: [11.7,4.33,-45]
#   location3: [14.4,-1.17,-90]
#   location5: [15.8,-1.53,60]
#   location4: [12.1,-6.84,90]  
#   outpose_location: [4.3,-4,30]
location:
  attack_place: [7.84,3.64]
  home: [-2.37,0]
  base: [12.9,3.49,140]
  recover_place: [-4.4,-6.3]
  home_xunluo: [-2.2,5.7]
  home_xunluo2: [-0.9,5.58]
  location1: [7.17,2.82,30]
  location2: [11.7,4.33,-45]
  location3: [14.4,-1.17,-90]
  location5: [15.8,-1.53,60]
  location4: [12.1,-6.84,90]  
  outpose_location: [4.3,-4,30]
